.circleci/config.yml

```diff
@@ -183,30 +218,72 @@ workflows:
 
   publish:
     jobs:
-      - docker/publish:
-          name: Push latest
-          before_build: *version
-          docker-password: DOCKER_PASS
-          docker-username: DOCKER_USER
-          image: mozilla/telemetry-airflow
-          tag: latest
+      - docker-build-artifact:
+          name: 🛠️ Docker build and persist image
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
```

Comment:
do the filters work as an &&? E.g does this mean for all tags but ignore all branches?

Reply:
They work as logical OR, this snippet is wrong. I`ll fix it. We want `branches` to be `only: main` this way we build and push for any tag OR any commit to `main`